content/about/jesp/index.md

```diff
@@ -26,7 +26,7 @@ The Jakarta EE Specification Committee hereby adopts the [Eclipse Foundation Spe
 4. A ballot will be declared invalid and concluded immediately in the event that the Specifiation Team withdraws from the corresponding review.
 5. Specification Projects must engage in at least one Progress or Release Review  per year while in active development.
 
-All development that modifies content in the `javax` namespace must be moved to a `jakarta` namespace. All `jakarta` namespace development must occur within the scope of a Specification Project operating under the purview of the Jakarta EE Working Group’s Specification Committee and must implement the process as defined by the most recently adopted revision of the JESP.
+All development that modifies content in the `javax` namespace must be moved to a `jakarta` namespace. All specifications must use the `jakarta` namespace. All `jakarta` namespace development must occur within the scope of a Specification Project operating under the purview of the Jakarta EE Working Group’s Specification Committee and must implement the process as defined by the most recently adopted revision of the JESP.
```

Comment:
Shouldn't this be more specific and clarify it's about API and not about other things, e.g. TCK? Something like:
"All API elements (Java types, properties, etc.) in specifications must use the `jakarta` namespace."